views/help/forms/form_wish.hbs

```diff
@@ -38,11 +38,11 @@
 	<textarea class="form-control" cols="50" name="benefit" rows="6" placeholder="{{$t "help.form_Wish.placeholder.toUpdateThem" }}" required></textarea>
 </label>
 <label class="form-group">
-	<span class="control-label optional">{{$t "help.form_Wish.label.acceptanceCriteria" }} {{$t "help.global.label.requiredField"}}</span>
+	<span class="control-label optional">{{$t "help.form_Wish.label.acceptanceCriteria" }}</span>
 	<textarea class="form-control" cols="50" name="acceptanceCriteria" rows="6" placeholder="{{$t "help.form_Wish.placeholder.afterLoggingOnToTheSystem" }}"></textarea>
 </label>
 <label class="form-group">
-	<span class="control-label optional">{{$t "help.form_Wish.label.device" }}</span>
+	<span class="control-label">{{$t "help.form_Wish.label.device" }} {{$t "help.global.label.requiredField"}}</span>
```

Comment:
This needs to be removed or wrapped by label-wrapper. Depending on the requirement